xarray/core/indexing.py

```diff
@@ -1379,14 +1385,22 @@ def __getitem__(self, key):
                     value = value[(slice(None),) * axis + (subkey,)]
                 return value
 
-    def __setitem__(self, key, value):
-        raise TypeError(
-            "this variable's data is stored in a dask array, "
-            "which does not support item assignment. To "
-            "assign to this variable, you must first load it "
-            "into memory explicitly using the .load() "
-            "method or accessing its .values attribute."
-        )
+    def __setitem__(self, key, newvalue):
```

Comment:
What's convention for `value` / `newvalue`?

Reply:
Didn't take notice. value is used elsewhere in xarray.